src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/PartiesNotified.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.sscs.model.single.hearing;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class PartiesNotified {
+
+    private Long partiesNotifiedRequestId;
```

Comment:
We should change this variable name to "hearingID" to match the response object in the swagger spec, see below.

```
{
  "hearingID": "string",
  "responses": [
    {
      "partiesNotified": "2022-05-12T20:42:09.623Z",
      "requestVersion": 0,
      "responseReceivedDateTime": "2022-05-12T20:42:09.623Z",
      "responseVersion": 0,
      "serviceData": {}
    }
  ]
}
```